### PR TITLE
Fernando Regalado - CSS HW1

### PR DESCRIPTION
**Pull request by fernandoregalado**
First attempt to update file in github.com

### CODE REVIEW

**Comment by fernandoregalado** [comment]
code and example: https://guides.github.com/features/mastering-markdown/ 
really useful!

**Comment by fernandoregalado** [comment]
really funny resource at https://www.webfx.com/tools/emoji-cheat-sheet/.

**Final comments:**
I have some prior experience using desktop github with python. Steps using R to commit and push are more complicated so I had to see the videos twice. However, It has the advantage that we can do all that stuff directly from R instead of openning first Spider (let say) and the Desktop Github to summit changes.

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
```
Comment (nearridge) [inline README.md:9]: I had no idea you could embed emojis in markdown. Thanks for teaching me something!

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
```
Comment (nearridge) [inline README.md:3]: Nice use of multilevel headers here.

File: README.md [hunk 3]
```diff
@@ -1 +1,15 @@
...
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
```
Comment (nearridge) [inline README.md:5]: Great job embedding this photo here and giving it a descriptive name

File: README.md [hunk 4]
```diff
@@ -1 +1,15 @@
...
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
+* :bank: Work: [*Central Reserve Bank of Peru*](https://www.bcrp.gob.pe/en)
+* Contact information:
+  * :id: @fernandoregalado
+  * :email: fernandoregalado@uchicago.edu
```
Comment (nearridge) [inline README.md:13]: You could even turn your email into a hyperlink just like with the Central bank of peru link!

**Review by nearridge — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |        x   |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Reproducibility**         |           |     x         |            |

Remarks: Nice job here Fernando! You made really effective use of the different stylistic elements of markdown and demonstrated mastery of the basic tools we'll be using in this class. I'm a little bit confused about your opening/closing of your pull request repeatedly though. Feel free to open your pull request once you've finished your assignment and are ready to submit. There's no reason to open things early and needing to re-open and close it.

File: README.md (hunk 2 above)
Comment (tyin29) [inline README.md:3]: Also a great job in changing the text color, which is something new I learned!

File: README.md [hunk 5]
```diff
@@ -1 +1,15 @@
...
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
+* :bank: Work: [*Central Reserve Bank of Peru*](https://www.bcrp.gob.pe/en)
+* Contact information:
```
Comment (tyin29) [inline README.md:11]: Maybe you can separate your contact information from education and work by making it a new header or just make it the same format as the first introduction.

File: README.md [hunk 6]
```diff
@@ -1 +1,15 @@
...
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
+* :bank: Work: [*Central Reserve Bank of Peru*](https://www.bcrp.gob.pe/en)
```
Comment (tyin29) [inline README.md:10]: Nice job to incorporate emoji in the beginning.

File: README.md [hunk 7]
```diff
@@ -1 +1,15 @@
...
+* Contact information:
+  * :id: @fernandoregalado
```
Comment (tyin29) [inline README.md:12]: I think you did well in making your id, email, and phone number a new unordered list, which makes the file looks more organized.

File: README.md (hunk 3 above)
Comment (tyin29) [inline README.md:5]: According to the link you inserted, I think the way to insert a picture from the local drive is to upload the picture to a folder in GitHub and then insert the link to the markdown. So you taught me how to insert a picture from the local drive.

File: README.md (hunk 2 above)
Comment (tyin29) [inline README.md:3]: But perhaps it would be better to keep your name in the black color.

File: README.md (hunk 5 above)
Comment (tyin29) [reply README.md:11]: Or maybe you can add an emoji before contact information to make it the format as education and work.

File: README.md (hunk 1 above)
Comment (tyin29) [inline README.md:9]: Maybe you don't need to make Harris School of Public Policy as well as Central Reserve Bank of Peru Italic. For me, keeping it in the original format looks better.

**Comment by tyin29** [comment]
Remarks:

Hi Fernando, You did a great job in editing the file and pushing the committed changes to GitHub. You included all elements mentioned in the assignment instruction and nicely arranged them. The coding manual was well followed and your presentation was carefully made to display important information (especially for the photo being well-sized). I really like your design, especially the insertion of emojis before each line below the introduction, which goes beyond my expectation of how this assignment should be completed. Also, I’m glad that you share the link to the emoji cheat sheet. I just added it to my bookmark. The only things I think you can improve on are just about the format. You can see my specific comments in the file.

File: README.md (hunk 7 above)
Comment (dvalverde123) [reply README.md:12]: I agree, this looked really cool!

**Comment by dvalverde123** [comment]
Hi Fernando, 

You did a really good job with this! I didn't know you could put emojis in R Markdown, so that was a new thing I learned! Your list with your contact info looked really nice and you did a good job with your heading. I also didn't know how to embed links, so I learned that too. I think one thing to think about in the future would be how many formatting and how many emojis you're putting into the file. I was really excited when I saw them, but my eye didn't know where to look first because there were so many. Also hello from a fellow Peruvian! I was born and raised in the US but my entire family is from Peru.

**Pull request closed without merging**